src/deadline/client/api/_submit_job_bundle.py

```diff
@@ -403,7 +403,7 @@ def wait_for_create_job_to_complete(
     )
 
 
-def _hash_attachments(
+def hash_attachments(
```

Comment:
I think this function has to be refactored and moved into Job Attachments code base. Sean leveraged this function for the JA CLIs.